src/gui/static/src/app/components/layout/hardware-wallet/hw-pin-dialog/hw-pin-dialog.component.html

```diff
@@ -19,19 +19,19 @@
   </div>
   <div class="select-address-theme">
     <div class="num-pad-row">
-      <button mat-button color="primary" (click)="addNumber('7')"><span>#</span></button>
-      <button mat-button color="primary" (click)="addNumber('8')"><span>#</span></button>
-      <button mat-button color="primary" (click)="addNumber('9')"><span>#</span></button>
+      <button mat-button color="primary" (click)="addNumber('7')"><span>•</span></button>
```

Comment:
can u define this as some sort of constant and than use that constant here instead?